src/modules/requirements/AttackEvolveRequirement.ts

```diff
@@ -11,7 +11,7 @@ export default class AttackEvolveRequirement extends Requirement {
     getProgress(): number {
         const partyPokemon = App.game.party.getPokemonByName(this.pokemon);
 
-        return partyPokemon?.attack >= pokemonMap[this.pokemon].attack * this.requiredValue ? 1 : 0;
+        return Math.min(Math.trunc(partyPokemon?.attack / pokemonMap[this.pokemon].attack), this.requiredValue);
     }
 
     hint(): string {
```

Comment:
Hint now returns `Level up to evolve` as soon as current attack >= base attack

Reply:
Good catch 👍